src/common/thumbnail-manager.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import {Utils, ThumbnailInfo, MediaType} from '@playkit-js/playkit-js';
+import evaluate from './utils/evaluate';
+
+const DefaultThumbnailConfig: Object = {
+  thumbsWidth: 164,
+  thumbsHeight: 92,
+  thumbsSlices: 100
+};
+
+const THUMBNAIL_REGEX = /.*\/p\/\d+\/(?:[a-zA-Z]+\/\d+\/)*thumbnail\/entry_id\/\w+\/.*\d+/;
+const THUMBNAIL_SERVICE_TEMPLATE: string = '{{thumbnailUrl}}/width/{{thumbsWidth}}/height/{{thumbsHeight}}/vid_slices/{{thumbsSlices}}/ks/{{ks}}';
+
+class ThumbnailManager {
+  _player: Player;
+  _thumbnailConfig: ?SeekbarConfig;
+
+  constructor(mediaConfig: KPMediaConfig, player: Player) {
+    this._player = player;
+    this._thumbnailConfig = this._buildKalturaThumbnailConfig(mediaConfig);
+  }
+
+  getThumbnail(time: number): ?ThumbnailInfo {
+    if (this._isUsingKalturaThumbnail()) {
+      return this._convertKalturaThumbnailToThumbnailInfo(time);
+    }
+    return this._player.getThumbnail(time);
+  }
+
+  getKalturaThumbnailConfig(): ?SeekbarConfig {
+    return this._thumbnailConfig;
```

Comment:
this is only for the ui until we change it to support the new api right?

Reply:
yes.